src/application/components/_applicationPreviewSubsection.scss

```diff
@@ -3,3 +3,21 @@
     margin: $spacing-l 0;
   }
 }
+
+// Selects the divs that are direct children of the root ApplicationPreviewSubsection,
+// excludes child ApplicationPreviewSubsections up to 4 levels deep
+.ApplicationPreviewSubsection:not(
```

Comment:
Is there any other easier way to define this? Isn't the purpose to add margin bottom for the last div or something? There is `:last-child` pseudo-element.

Reply:
The purpose is to add the margin-bottom to .ApplicationPreviewSubsection classes that are in the "top level". Your approach might work or not, I don't remember what I tried. The issue I had is that these subsections render more subsections that render more subsections, etc.
And I want to apply the style to the all "top level" classes only.

Do you have any suggestions? I don't know of a simple solution, which is why it looks like this here :smile:

Reply:
With the updated form, it doesn't even seem to be working correctly. So how to design this for all possible recursive cases, and forms that are present or hidden, but might have answers or not? :thinking: